### PR TITLE
#10825 Fixed `yii\validators\EachValidator` does not respect `skipOnEmpty` when using model

### PR DESCRIPTION
**Pull request by webdevsega**
| Q | A |
| --- | --- |
| Is bugfix? | yes |
| New feature? | no |
| Breaks BC? | no |
| Tests pass? | yes |
| Fixed issues | #10825 |

### CODE REVIEW

File: framework/validators/EachValidator.php [hunk 1]
```diff
@@ -129,7 +129,9 @@ public function validateAttribute($model, $attribute)
         $filteredValue = [];
         foreach ($value as $k => $v) {
             $model->$attribute = $v;
-            $validator->validateAttribute($model, $attribute);
+            if (!$validator->skipOnEmpty || !$validator->isEmpty($v)) {
```
Comment (SilverFire) [inline framework/validators/EachValidator.php]: It will be easier to read the following condition:

`!($validator->skipOnEmpty && $validator->isEmpty($v)`

**Comment by SilverFire** [comment]
Could you add a changelog line please?

**Comment by webdevsega** [comment]
@SilverFire yes, but in changelog already has a line associated with this bug in 2.0.7 version

**Comment by SilverFire** [comment]
@klimov-paul could you verify this PR?

**Comment by klimov-paul** [comment]
Thank you.

**Comment by klimov-paul** [comment]
CHANGELOG updated by b5ff914d809dea0a532f5fbb4640956f328ab9f9

**Pull request merged**